src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentList = () => (
-  <>Put the list here</>
-);
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export function CommentList({ comments }) {
+  const hasComments = comments.length;
```

Comment:
I believe if length is gonna be 0 it's not gonna work correctly, to make sure everything will work fine - convert it to Boolean, by uising !! or Boolean